### PR TITLE
Added notes about Secure Servers

### PR DESCRIPTION
**Pull request by skabashnyuk**
### What does this PR do?
Added notes about Secure Servers

### What issues does this PR fix or reference?
https://github.com/eclipse/che-docs/pull/457
https://github.com/eclipse/che/issues/10762

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: src/main/pages/release-notes/release-6.10.0.md [hunk 1]
```diff
@@ -22,13 +22,25 @@ Instruction how to upgrade
 
 ## Release Details
 
-### Feature1 (#ISSUE)
+### Secure Servers (#ISSUE)
```
Comment (slemeur) [inline src/main/pages/release-notes/release-6.10.0.md]: Servers in our terminology is having a particular meaning, which might not be known and understood by external audience. 
What about "Secure Workspace Tooling Communication" ?

File: src/main/pages/release-notes/release-6.10.0.md [hunk 2]
```diff
@@ -22,13 +22,25 @@ Instruction how to upgrade
...
+Eclipse Che extension developers might be interested in a new feature - "Secure Servers"
+It can be useful if you want to protect your web-based tool, that running as part of a workspace with Che authentification.
```
Comment (slemeur) [inline src/main/pages/release-notes/release-6.10.0.md]: How does it work?

Comment (slemeur) [reply src/main/pages/release-notes/release-6.10.0.md]: In what case, I don't want to secure my tools?

**Review by slemeur — COMMENTED** [review]
Thanks for starting that! Nice to see that coming along. 
I'm giving feedback related to the perspective on how this should be oriented for external audience?

File: src/main/pages/release-notes/release-6.10.0.md (hunk 1 above)
Comment (skabashnyuk) [reply src/main/pages/release-notes/release-6.10.0.md]: We do have this terminology in our docs. https://www.eclipse.org/che/docs/servers.html
And this is oriented for power eclipse che extension developers, not end users.

File: src/main/pages/release-notes/release-6.10.0.md (hunk 2 above)
Comment (skabashnyuk) [reply src/main/pages/release-notes/release-6.10.0.md]: off by default. Details of configuration in a separate doc. https://github.com/eclipse/che-docs/pull/457

File: src/main/pages/release-notes/release-6.10.0.md [hunk 3]
```diff
@@ -22,13 +22,26 @@ Instruction how to upgrade
...
+    "port": "4921",
+    "protocol": "http",
+    "attributes": {
+        "secure": "true",
+        "unsecuredPaths": "/liveness",
+        "cookiesAuthEnabled": "true"
+    }
+}
+```
+TODO link to docs
```
Comment (sleshchenko) [inline src/main/pages/release-notes/release-6.10.0.md]: FYI The corresponding PR is merged https://github.com/eclipse/che-docs/pull/457
After site update it should be available by the following link https://www.eclipse.org/che/docs/servers.html#secure-servers

File: src/main/pages/release-notes/release-6.10.0.md (hunk 1 above)
Comment (slemeur) [reply src/main/pages/release-notes/release-6.10.0.md]: The release note is not targetting "power eclipse che extension developers" but all users of Eclipse Che who are interested to learn what's new. The language must be appropriate to an audience without too deep knowledge of Che.

File: src/main/pages/release-notes/release-6.10.0.md (hunk 2 above)
Comment (slemeur) [reply src/main/pages/release-notes/release-6.10.0.md]: Problems and solutions oriented.
A user will ask himself: "Why should I secure the communication between servers?" and "Why should I not do it?"

Having the capability is nice - but if user don't understand why he should leverage it then this is not helpful for him...

Comment (skabashnyuk) [reply src/main/pages/release-notes/release-6.10.0.md]: I don't understand what I shall do with that.

**Pull request merged**